docs/api.md

```diff
@@ -1418,6 +1418,16 @@ the event will be called `"chat:name"`, with name being the name passed
   * `repeat` - defaults to true, whether to listen for this event after the first match
   * `parse` - instead of returning the actual message that was matched, return the capture groups from the regex
 
+#### bot.removeChatPattern(name)
+
+removes a chat pattern
+* `name` the name of the chat pattern
+
+#### bot.removeChatPatternSet(name)
```

Comment:
Am I mistaken or do you have two APIs that do the same thing?

Reply:
They do the same thing but I can remove 1